cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -61,15 +61,30 @@ func NormalizeFlags(f *pflag.FlagSet, name string) pflag.NormalizedName {
 }
 
 func SelectFieldMask(cmdFlags *pflag.FlagSet, fieldMaskFlags ...*pflag.FlagSet) (paths []string) {
+	all := false
 	cmdFlags.Visit(func(flag *pflag.Flag) {
 		flagName := toUnderscore.Replace(flag.Name)
+		if flagName == "all" {
+			all = true
+			return
+		}
 		for _, fieldMaskFlags := range fieldMaskFlags {
 			if b, err := fieldMaskFlags.GetBool(flagName); err == nil && b {
 				paths = append(paths, flagName)
 				return
 			}
 		}
 	})
+
+	if all {
```

Comment:
Why do we need a separate loop here? Can't we just add an "or all" to the `if` in the loop above?

Reply:
Improved handling of `--all` flag, using the same loop